### PR TITLE
Cecilia/upstream/pdpd frontend

### PR DESCRIPTION
**Pull request by ceciliapeng2011**
### Details:
 - *item1*
 - *...*

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: ngraph/test/frontend/paddlepaddle/op.cpp [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+static FrontendOpTestParam conv2d() {
+    FrontendOpTestParam res;
+    res.m_frontEndName = PDPD;
+    res.m_modelsPath =   PATH_TO_MODELS;
+    res.m_modelName =    "conv2d/";  //TODO: compact model/decomposited
+
+    //Inputs inputs;
+    // data (1, 3, 4, 4) input tensor
+    res.inputs.emplace_back(test::NDArray<float, 4>{{{{{0.f, 1.f, 2.f, 3.f, 4.f},
```
Comment (nosovmik) [inline ngraph/test/frontend/paddlepaddle/op.cpp:23]: I'm getting exception here, looks like input needs 1x3x4x4=48 elements, but only 35 is given

File: ngraph/test/frontend/shared/src/op.cpp [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+#include "../include/op.hpp"
+
+using namespace ngraph;
+using namespace ngraph::frontend;
+
+using TestEngine = test::IE_CPU_Engine;
+
+std::string FrontendOpTest::getTestCaseName(const testing::TestParamInfo<FrontendOpTestParam> &obj) {
+    std::string res = obj.param.m_frontEndName + "_" + obj.param.m_modelName;
+    //res += "I" + joinStrings(obj.param.m_oldInputs) + joinStrings(obj.param.m_newInputs);
```
Comment (nosovmik) [inline ngraph/test/frontend/shared/src/op.cpp:15]: it's fine to have it this way for now, just note that generated 'testCaseName' must be unique for test system.
Combining it only with frontendName and modelName, you'll not be able to run FrontendOptest with same model, but with different inputs

File: ngraph/test/frontend/shared/src/op.cpp [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    // load
+    ASSERT_NO_THROW(m_fem.availableFrontEnds());
+    ASSERT_NO_THROW(m_frontEnd = m_fem.loadByFramework(m_param.m_frontEndName));
+    ASSERT_NE(m_frontEnd, nullptr);
+    ASSERT_NO_THROW(m_inputModel = m_frontEnd->loadFromFile(m_param.m_modelName));
+    ASSERT_NE(m_inputModel, nullptr);
+
+    // convert
+    std::shared_ptr<ngraph::Function> function;
+    function = m_frontEnd->convert(m_inputModel);
```
Comment (nosovmik) [inline ngraph/test/frontend/shared/src/op.cpp:42]: ASSERT_NO_THROW

File: ngraph/test/frontend/paddlepaddle/op/pool2d.cpp [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+    return res;
+}
+
+TEST_P(pool2dTest, test_pool2d) {
+    validateOp();
+}
+
+INSTANTIATE_TEST_CASE_P(FrontendOpTest, pool2dTest,
+                        ::testing::Values(
+                            avgPool(),
```
Comment (nosovmik) [inline ngraph/test/frontend/paddlepaddle/op/pool2d.cpp:89]: I'm getting "Unsupported pooling type" here, is it expected?

**Comment by ceciliapeng2011** [comment]
close it as it is obsolete.

**Pull request closed without merging**